nni/tools/nnictl/config_schema.py

```diff
@@ -73,16 +69,13 @@ def __init__(self, algo_type):
         }
         self.builtin_name_schema = {}
         for k, n in self.builtin_keys.items():
-            self.builtin_name_schema[k] = {Optional(n): setChoice(n, *get_all_builtin_names(k+'s'))}
+            self.builtin_name_schema[k] = {Optional(n): setType(n, str)}
```

Comment:
why converting from `setChoice` to `setType`?

Reply:
Because `get_all_builtin_names` no longer works. And since this is V1 config only, I believe it's not worthy to fix it.